examples/v2/chat2.nim

```diff
@@ -139,7 +166,10 @@ proc writeAndPrint(c: Chat) {.async.} =
     else:
       # XXX connected state problematic
       if c.started:
-        c.publish(line)
+        # Get message timestamp
+        let time = getTime().utc().format("'<'HH:mm'>'")
```

Comment:
Mmm. I agree, but was worried that it might clutter the chat space if every message is prepended by a timestamp, date and nick. One idea is to have some kind of "day" divider between different dates, but that may require more sophisticated state-keeping than is necessary in a simple chat app. I might just try prepending the shortest date format possible to every message and see how it affects UX 😄, e.g.
`<Mar 3, 7:15> nick: this is a chat message`